### PR TITLE
integration/env, integration/setup: update env vars

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/5278.

Deviations from aws-operator:

- `E2E_` prefixes for e2e specific variables. This is to reduce mess in circle.
- `E2E_TEST_DIR` is required.
- No version bundle stuff.
- No wip/cur.

### CODE REVIEW

File: integration/env/common.go [hunk 1]
```diff
@@ -3,57 +3,72 @@ package env
...
 	registryPullSecret = os.Getenv(EnvVarRegistryPullSecret)
 	if registryPullSecret == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarRegistryPullSecret))
 	}
 }
 
-func CircleCI() string {
-	return circleCI
+func CircleCI() bool {
+	return circleCI == strings.ToLower("true")
```
Comment (xh3b4sd) [inline integration/env/common.go]: Shouldn't we rather use https://golang.org/pkg/strconv/#ParseBool? Also, `true` is already "to lower".

Comment (kopiczko) [reply integration/env/common.go]: Good call. I didn't know it exists. Also it should be `strings.ToLower(circleCI)` LoL.

Comment (kopiczko) [reply integration/env/common.go]: Problem with using strconv.ParseBool is it will fail on empty string and we need to handle an error.

Comment (kopiczko) [reply integration/env/common.go]:
```suggestion
	return strings.ToLower(circleCI) =="true"
```

**Pull request merged**